src/store/actions.js

```diff
@@ -261,69 +311,108 @@ export default {
-		// 		Vue.prototype.$chamaileon.htmlImport = null;
-		// 		commit("setHtmlImportInited", false);
-		// 	}
-		// }
+	async initHtmlImport({ getters, commit, state, dispatch }) {
+		if (state.htmlImportInited === true || state.htmlImportInited === "pending") return;
+		commit("setHtmlImportInited", "pending");
+		await dispatch("waitForSdkToBeInited");
+		try {
+			Vue.prototype.$chamaileon.htmlImport = await Vue.prototype.$chamaileon.createFullscreenPlugin({
+				...getters.getHtmlImportConfigObject,
+				plugin: "import",
+				hooks: {
+					cancel: () => {
+						Vue.prototype.$chamaileon.htmlImport.hide();
+					},
+					close: () => {
+						Vue.prototype.$chamaileon.htmlImport.hide();
+					},
+					onReplaceImages: async (imageSrcArray) => {
+						if (!imageSrcArray || imageSrcArray.length === 0) {
+							return null;
+						}
+						try {
+							// purging duplicates and nullies
+							const imageSrcArrayFiltered = [ ...new Set(imageSrcArray) ].filter(x => !!x);
+
+							const imagesArrayReplaced = await Promise.allSettled(imageSrcArrayFiltered.map((imageSrc) => {
+								// eslint-disable-next-line no-async-promise-executor
+								return new Promise(async (resolve, reject) => {
+									try {
+										const imageIsOnOurDomain = /(https?:\/\/(.+?\.)?(chamaileon\.io|plugins\.edmdesigner\.com)(\/[A-Za-z0-9\-\._~:\/\?#\[\]@!$&'\(\)\*\+,;\=]*)?)/.test(imageSrc);
+										if (imageIsOnOurDomain) {
+											return resolve({
+												oldSrc: imageSrc,
+												newSrc: imageSrc,
+											});
+										}
+
+										const dataUri = await processFileupload(imageSrc);
+
+										if (!dataUri) {
+											return null;
+										}
+
+										const imageName = imageSrc.substr(imageSrc.lastIndexOf("/") + 1);
+										const _id = Date.now() + Math.random();
+										const newImageData = {
+											name: decodeURIComponent(imageName),
+											parentId: "root",
+											src: dataUri,
+											createdAt: new Date(),
+											_id,
+										};
+
+										await images.insert([ newImageData ]);
+
+										return resolve({
+											oldSrc: imageSrc,
+											newSrc: dataUri,
+										});
+									} catch (e) {
+										console.error(e);
+										reject(imageSrc);
+									}
+								});
+							}));
+							const responseArray = imagesArrayReplaced.map((response) => {
+								if (response.status === "rejected") {
+									return {
+										oldSrc: response.reason,
+										newSrc: response.reason,
+									};
+								}
+								return response.value;
+							});
+
+							return responseArray;
+						} catch (e) {
+							console.error(e);
+							return null;
+						}
+					},
+					onImportReady: async (result) => {
+						const document = { ...result.document, title: "Imported email" };
+						Vue.prototype.$chamaileon.htmlImport.hide();
+						await Vue.prototype.$chamaileon.emailEditor.methods.updateData({ document });
```

Comment:
```suggestion
						await Vue.prototype.$chamaileon.emailEditor.methods.updateData({ document });
						await Vue.prototype.$chamaileon.htmlImport.hide();
```
ebben a sorrendben nem fog teljesen összeakadni a két plugin szerintem. de biztos nem vagyok benne